lib/autoload/duplicator.rb

```diff
@@ -36,7 +36,8 @@ def duplicate(stuff)
     duplicated_stuff = []
     stuff = [*stuff] unless stuff_is_enumerable
     stuff.each do |obj|
-      duplicated_stuff << duplicate_object(obj)
+      duplicated_object = duplicate_object(obj)
+      duplicated_stuff << duplicated_object unless duplicated_object.nil?
```

Comment:
@jeremyyap just a quick question, this was changed because it is possible for `duplicate_object(obj)` to return nil?